config/locales/simple_form.zh-CN.yml

```diff
@@ -3,11 +3,14 @@ zh-CN:
   simple_form:
     hints:
       defaults:
-        avatar: 最大 2MB，限 PNG, GIF 或 JPG 格式，将缩到 120x120px
+        avatar: 最大 2 MB，限 PNG、GIF 或 JPG 格式，将缩到 120x120 px
+        digest: 在你长时间未登录的情况下，我们会向你发送一份摘要，其中含有提及你的嘟文的简要信息
         display_name: 不起过 30 个字符
-        header: 最大 2MB，限 PNG, GIF 或 JPG 格式，将缩到 700x335px
+        header: 最大 2 MB，限 PNG、GIF 或 JPG 格式，将缩到 700x335 px
```

Comment:
太棒了！顿号！（激动握手）终于找到会用顿号的人了！
建议把 `x` 变成乘号 `×`。

单位前加空格加太多的话可能不是太爽……可以看看英文原文怎么用的。（反正中英文都是”建议有距离，实际可选“。）

Reply:
+1 for `×`. As I said at the beginning I was following 3rd-party guidelines, since we don't have a clear one at here. I'll wait for others' ideas on those spaces :)